definitions/ext-snowflake_account/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: SNOWFLAKE_ACCOUNT
```

Comment:
I just want to double check this makes sense: 

the reference to `_ACCOUNT` seems pretty odd to me? Would this be better classed as `SNOWFLAKE`? 
eg no other type has a reference to account, even though the domainID could technically be an accountId.

Or is this fine?

Reply:
@cianBuckley the idea is to have two entities synthesised from Snowflake. One is a `SnowflakeAccount` which holds metrics about your Snowflake account such as how many DB's, how many warehouses, billing etc.

The other entity will be a `SnowflakeVirtualWarehouse` and this entity contains metric associated with the compute layer of snowflake and therefore has metrics about query performance etc.

So by saying the `TYPE: SNOWFLAKE_ACCOUNT` I was attempting to differentiate between these two entities. Does this make sense?

Reply:
Note: This PR is only for the entity SnowflakeAccount. If this goes well and I understand the process, I'll submit the 2nd one in Jan.

Reply:
thanks